en/includes/guides/authentication/saml/federated-idp-initiated-sso.md

```diff
@@ -0,0 +1,219 @@
+
+    ![Add external IdP as a sign-in option]({{base_path}}/assets/img/guides/applications/saml-app/add-federated-sso-sign-in-option.png){: width="500" style="border: 0.3px solid lightgrey;"}
+
+9. Click **Update** to save the changes.
+
+### Register the internal IdP as an application
+
+The internal IdP needs to be registered as an application in the external IdP. This enables the internal IdP to consume the SAML assertion sent by the external IdP after user authentication.
+
+To do so, follow the steps below in the external IdP.
+
+1. On the {{product_name}} Console, go to **Applications**.
+
+2. Click on **New Application** and select **Standard-Based Application**.
+
+3. Provide `Internal` as the name and select **SAML** as the protocol.
+
+3. Configure the following:
+
+    <table>
+        <thead>
+            <th>Parameter</th>
+            <th>Description</th>
+            <th>Value</th>
+        </thead>
+        <tr>
+            <td>Issuer</td>
+            <td>The ID of the application. In this instance, this application consumes the SAML assertion sent by the external IdP and therefore should match the <code>Service Provider Entity ID</code> that was configured when registering the external IdP as a connector (steps in the section above).</td>
+            <td><code>Internal</code></td>
+        </tr>
+        <tr>
+            <td>Assertion consumer service URLs</td>
+            <td>This is the URL that consumes the assertion sent by the IdP.</td>
+            <td><code>https://localhost:9443/commonauth</code></td>
+        </tr>
+    </table>
+
+4. Click **Create**.
+
+5. Go to the **Protocol** tab of the created application and under **Single Sign-On Profile**, select the **Enable IdP initiated SSO** checkbox.
+
+6. Click **Update** to save the changes.
+
+## Try it out
+
+You have now made the application in your internal IdP accessible to the users belonging to the external IdP. Follow the steps below to try out logging into the internal application as an external user.
+
+### Deploy the application
+
+Follow the steps below to download and run the `travelocity.com` application:
+
+!!! note "Before you begin"
+
+    - Make sure you have [Apache Tomcat 8.x.x](https://tomcat.apache.org/download-80.cgi). The  `webapps` directory in the Tomcat installation will be referred to as `<WEBAPPS>`.
+
+    - It is recommended to change the hostname to something other than `localhost` to avoid browser errors. Modify your machine's `/etc/hosts` entry to reflect this.
+
+1. Download the [sample application](https://github.com/wso2/samples-is/releases/download/v4.6.2/travelocity.war){: target="_blank"}
+
+2. Copy the `travelocity.war` file into `<WEBAPPS>`.
+
+    !!! note
+        In order to be consistent with the configurations above, be sure to rename the file as `travelocity.com.war`.
+
```

Comment:
Also need to change the following configs in the <WEBAPPS>/travelocity.com/WEB-INF/classes/travelocity.properties
```
SAML2.EnableSLO=false
SAML2.EnableResponseSigning=false
```